setup.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
These should be moved into a `Setup` controller.

The `Index` controller will need to be modified to have a `Base` that shares what the `__construct` does in `Index`.